ZenPacks/zenoss/LinuxMonitor/zenpack.yaml

```diff
@@ -239,6 +239,107 @@ device_classes: !ZenPackSpec
                                 rrdmin: 0
                             missing:
                                 rrdmin: 0
+                graphs:
+                    Utilization:
+                        units: percent
+                        miny: 0
+                        maxy: 100
+                        graphpoints:
+                            size:
+                                dpName: volume_size
+                                lineType: DONTDRAW
+                            free:
+                                dpName: volume_free
+                                lineType: AREA
+                                rpn: "size,-,size,/,-100,*"
+                thresholds:
+                    unallocatable:
+                        dsnames: [status_allocatable]
+                        minval: 1
+                        eventClass: /Status
+                    exported:
+                        dsnames: [status_exported]
+                        maxval: 0
+                        severity: 2
+                        eventClass: /Status
+                    missing:
+                        dsnames: [status_missing]
+                        maxval: 0
+                        eventClass: /Status
+            VolumeGroup:
+                description: Volume Group Monitoring
+                targetPythonClass: ZenPacks.zenoss.LinuxMonitor.VolumeGroup.VolumeGroup
+                datasources:
+                    group:
+                        type: COMMAND
+                        commandTemplate: '/usr/bin/sudo /usr/sbin/vgs --noheadings --units b --nosuffix -o vg_name,vg_size,vg_free'
```

Comment:
`vgs` is not in /usr/sbin at least on Ubuntu. I would recommend removing the absolute path and let the system's natural path find it instead.